deploy/osd-pcap-collector/04-pcap-dedicated-admins-scc.yaml

```diff
@@ -1,13 +0,0 @@
-kind: SecurityContextConstraints
```

Comment:
This SCC should not be deleted as this is needed for customers running as dedicated-admins to collect pcaps.